server/graphql/v1/mutations/collectives.js

```diff
@@ -316,7 +316,7 @@ export async function createCollectiveFromGithub(_, args, req) {
   const host = await models.Collective.findByPk(defaultHostCollective('opensource').CollectiveId);
   const promises = [
     collective.addUserWithRole(user, roles.ADMIN),
-    collective.addHost(host, user),
+    collective.addHost(host, user, { activity: activities.COLLECTIVE_CREATED_GITHUB }),
```

Comment:
Be clearer about the intent of the option, just pass something like `skipApplyActivity`.